release-notes/9.0/9.0.0/api-diff/Microsoft.NETCore.App/9.0.0_System.Collections.Generic.md

```diff
@@ -0,0 +1,214 @@
++            bool System.Collections.ICollection.IsSynchronized { get; }
++            object System.Collections.ICollection.SyncRoot { get; }
++            bool System.Collections.IList.IsFixedSize { get; }
++            bool System.Collections.IList.IsReadOnly { get; }
++            object? System.Collections.IList.this[int index] { get; set; }
++            public void CopyTo(TValue[] array, int arrayIndex);
++            public OrderedDictionary<TKey, TValue>.ValueCollection.Enumerator GetEnumerator();
++            void ICollection<TValue>.Add(TValue item);
++            void ICollection<TValue>.Clear();
++            bool ICollection<TValue>.Contains(TValue item);
++            bool ICollection<TValue>.Remove(TValue item);
++            IEnumerator<TValue> IEnumerable<TValue>.GetEnumerator();
++            int IList<TValue>.IndexOf(TValue item);
++            void IList<TValue>.Insert(int index, TValue item);
++            void IList<TValue>.RemoveAt(int index);
++            void System.Collections.ICollection.CopyTo(Array array, int index);
++            IEnumerator System.Collections.IEnumerable.GetEnumerator();
++            int System.Collections.IList.Add(object value);
++            void System.Collections.IList.Clear();
++            bool System.Collections.IList.Contains(object value);
++            int System.Collections.IList.IndexOf(object value);
++            void System.Collections.IList.Insert(int index, object value);
++            void System.Collections.IList.Remove(object value);
++            void System.Collections.IList.RemoveAt(int index);
++            public struct Enumerator : IDisposable, IEnumerator, IEnumerator<TValue> {
++                public TValue Current { get; }
++                object? System.Collections.IEnumerator.Current { get; }
++                public bool MoveNext();
++                void System.Collections.IEnumerator.Reset();
++                void System.IDisposable.Dispose();
++            }
++        }
++    }
+     public class PriorityQueue<TElement, TPriority> {
++        public bool Remove(TElement element, out TElement removedElement, out TPriority priority, IEqualityComparer<TElement>? equalityComparer = null);
+         public sealed class UnorderedItemsCollection : ICollection, IEnumerable, IEnumerable<(TElement, TPriority)>, IReadOnlyCollection<(TElement, TPriority)> {
+-            IEnumerator<(TElement Element, TPriority Priority)> IEnumerable<(TElement, TPriority)>.GetEnumerator();
+
++            IEnumerator<(TElement Element, TPriority Priority)> IEnumerable<(TElement, TPriority)>.GetEnumerator();
+             public struct Enumerator : IDisposable, IEnumerator, IEnumerator<(TElement, TPriority)> {
+-                (TElement Element, TPriority Priority) IEnumerator<(TElement, TPriority)>.Current { get; }
+
++                (TElement Element, TPriority Priority) IEnumerator<(TElement, TPriority)>.Current { get; }
```

Comment:
nothing seems to have changed here, and note the empty lines not present anywhere else (for some reason which could be related to this zero diff)